### PR TITLE
added flappyETH

### PR DESCRIPTION
**Pull request by jsnx**
### Checklist
- [x] Token Address in lowercase
- [ ] Link to verified contract on Blastscan
- [x] Description of token if non-standard ERC20, e.g. has tax, OFT token, bridged XERC20, ERC-404, etc.
- [x] An SVG <150 KB in size, cropped to be a circle. **Rectangles and other shapes of images will not be accepted.**
- [x] Token category you would like to be listed under
    - Current categories: DeFi, Memecoin, Stablecoin, Game, Other
- [x] Ran the `update_list.sh` script to generate the `token-list.json` file **DO NOT MANUALLY EDIT THE FILE**

### Additional Information
-

### CODE REVIEW

File: chains/81457/assets/0x1BB5E4Dd6F18E8941fCe273379DF2223cD08C2da/token-info.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "tokenAddress": "0x1BB5E4Dd6F18E8941fCe273379DF2223cD08C2da",
```
Comment (jooleseth) [inline chains/81457/assets/0x1BB5E4Dd6F18E8941fCe273379DF2223cD08C2da/token-info.json]: needs to be lowercase

File: chains/81457/token-list.json [hunk 2]
```diff
@@ -69,6 +69,13 @@
     "tokenDecimals": 18,
     "tokenCategory": "Game"
   },
+  {
```
Comment (jooleseth) [inline chains/81457/token-list.json]: script not ran correctly

**Comment by jsnx** [comment]
Check again plz. Changed to lowercase and ran scrip again.

File: chains/81457/assets/0x1BB5E4Dd6F18E8941fCe273379DF2223cD08C2da/token-info.json
Comment (jooleseth) [inline chains/81457/assets/0x1BB5E4Dd6F18E8941fCe273379DF2223cD08C2da/token-info.json:1]: folder path is not correct lowercase

**Comment by jsnx** [comment]
check again please

**Review by jooleseth — APPROVED** [review]

**Pull request merged**